build.gradle

```diff
@@ -339,19 +369,20 @@ task compressTiny(dependsOn: ["tinyJar", "mergeTiny"], type: FileOutput){
 		fileInputStream.close()
 		outputStream.finish()
 		outputStream.close()
-		inputFile.delete()
 	}
 }
 
 clean.doFirst {
 	delete tempDir, cacheFilesMinecraft
 }
 
+tasks.build.dependsOn "checkMappings"
```

Comment:
no, this should always be ran as a separate task and should not stop build from working.

Reply:
Fixed locally.